routes/web.php

```diff
@@ -14,3 +14,27 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Auth::routes();
+
+Route::get('/home', 'HomeController@index')->name('home');
+
```

Comment:
Where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=339395368 Accessing The Current Route

Reply:
i use for feature sidebar.